.devcontainer/xdebug.ini

```diff
@@ -0,0 +1,7 @@
+zend_extension=xdebug.so
+xdebug.mode=debug,coverage
+xdebug.start_with_request=yes
+xdebug.client_host=host.docker.internal
+xdebug.client_port=9003
+xdebug.idekey=PHPSTORM
```

Comment:
We should probably leave that to the `DBGP_IDEKEY` environment setting

Reply:
I don't know how to deal with this comment, can you help me?

Reply:
This line is hardcoded for PhpStorm (I am not sure whether that IDE is used at all for running dev containers, by the way). Other values include `VSCODE`, etc. So a more configurable approach is via the environment variable `DBGP_IDEKEY`
See https://xdebug.org/docs/all_settings#idekey
And the environment variable could be defined in a `.env` (either be default or by the user) https://code.visualstudio.com/remote/advancedcontainers/environment-variables#_option-2-use-an-env-file

But it does not look super important, so maybe for now just removing this line should be enough, and we can come back to it when writing the documentation.